rb_ws/src/buggy/scripts/auton/autonsystem.py

```diff
@@ -53,6 +53,7 @@ def __init__(self, trajectory, controller, brake_controller, buggy_name, is_sim)
             rospy.Subscriber(buggy_name + "/nav/odom", Odometry, self.update_msg)
 
 
+        rospy.Subscriber("nav/odom", Odometry, self.update_msg)
```

Comment:
Should this be in an else statement?

Reply:
unsure, i don't believe i edited this file

Reply:
unsure, i don't believe i edited this file?